package.json

```diff
@@ -15,19 +15,19 @@
     "build": "kolibri-tools build prod --file ./build_tools/build_plugins.txt && yarn run hashi-build",
     "makemessages": "kolibri-tools build i18n --file ./build_tools/build_plugins.txt",
     "watch": "kolibri-tools build dev --file ./build_tools/build_plugins.txt",
-    "watch-hot": "kolibri-tools build dev --file ./build_tools/build_plugins.txt --hot",
+    "watch-hot": "yarn run watch --hot",
     "django-devserver": "kolibri --debug manage runserver --settings=kolibri.deployment.default.settings.dev \"0.0.0.0:8000\" ",
-    "devserver": "npm-run-all --parallel django-devserver watch lint:watch hashi-dev",
-    "devserver-hot": "npm-run-all --parallel django-devserver watch-hot lint:watch",
+    "devserver": "npm-run-all --parallel django-devserver fmt-frontend:watch fmt-backend:watch hashi-dev watch",
+    "devserver-hot": "npm-run-all --parallel django-devserver fmt-frontend:watch fmt-backend:watch hashi-dev watch-hot",
     "bundle-stats": "kolibri-tools build stats --file ./build_tools/build_plugins.txt",
     "clean": "kolibri-tools build clean --file ./build_tools/build_plugins.txt",
     "preinstall": "node ./packages/kolibri-tools/lib/npm_deprecation_warning.js",
-    "lint": "kolibri-tools lint",
-    "lint-all": "yarn run lint '{kolibri*/**/assets,packages}/**/*.{js,vue,scss,less,css}' -i '**/node_modules/**','**/static/**','**/packages/kolibri-core-for-export/**'",
-    "lint-all:fix": "yarn run lint-all -w",
-    "lint:fix": "yarn run lint -w",
-    "lint:watch": "yarn lint-all -w -m",
-    "generate-locale-data": "node ./build_tools/i18n/intl_code_gen.js",
+    "fmt-frontend": "yarn run fmt-frontend:check --write",
```

Comment:
Note that this command doesn't just format, it also runs linting and errors if linting has failed, so I am not sure this name change is appropriate for the frontend.

Reply:
yes... on the other hand it doesn't just lint, it also formats :)

I went with `fmt` because on the backend there are other places that _also_ lint (e.g. flake8), and I wanted to distinguish from that, and because 'lint' seems a bit jargon-y. (It's not used on either the Prettier or Black project pages)

Reply:
Right, on the backend, definitely, Black is only formatting, but that's not the case for the frontend.

Given that eslint and stylelint also autofix things I don't see that `linting` is a bad general term for the format + warn behaviour.

Reply:
¯\_(ツ)_/¯ yeah my strategy was to keep the verbs the same for the sake of [simpler docs](https://github.com/learningequality/kolibri/pull/5333/files#diff-9bc8bf6e8d9db020cefce1fbc0426795R359) but if `fmt` feels misleading to you I'll switch it back

Reply:
Note that below, I changed the semantics of the command a bit for consistency:

`fmt-frontend": "yarn run fmt-frontend:check --write",
    "fmt-frontend:check    "fmt-frontend:watch":

Reply:
Note that below, I changed the semantics of the command a bit for consistency:

* `fmt-thing` formats and lints
* `fmt-thing:check` just lints
* `fmt-thing:watch` watches, formats, and lints

so I might revert that change too

Reply:
reverted changes here: https://github.com/learningequality/kolibri/pull/5333/commits/0fb396a16ebd89d24bd5ef8cdcb6ea8caac774a7